### PR TITLE
fix(2954): Modify Artifact preview

### PR DESCRIPTION
**Pull request by kumada626**
## Context

Modify the Artifact preview page layout.

Before:
- There is a blank space under the artifact preview window.
- Artifact tree cannot scroll horizontally. Because of this, deep directory paths or long file names are cutoff.
before_artifact_preview

After:
after_artifact_preview_2

## Objective

Set proper properties on artifact preview iframe and artifact-tree-content.

## References

https://github.com/screwdriver-cd/screwdriver/issues/2954

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/artifact-preview/style.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+& {
+  iframe {
+    width: 100%;
+    height: calc(90vh - 2rem);
```
Comment (kumada626) [inline app/components/artifact-preview/style.scss:4]: In the artifact preview page, there is a download button above the preview window. So I subtracted a button size.

**Comment by itleigns** [comment]
Can we make the button to switch between Steps and Artifacts static, so it doesn't scroll?

**Comment by kumada626** [comment]
@itleigns Fix.
after_artifact_preview_2

**Review by itleigns — APPROVED** [review]
Thank you!

**Review by jithine — APPROVED** [review]

**Pull request merged**